package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mapbox/node-pre-gyp",
   "description": "Node.js native addon binary install tool",
-  "version": "1.0.10",
+  "version": "1.0.11-dev.1",
```

Comment:
lgtm. Thanks. Could you finalise the package version.

Reply:
Done. Please re-approve the PR. Thanks.